docs/src/main/sphinx/release/release-453.md

```diff
@@ -0,0 +1,135 @@
+# Release 453 (25 Jul 2024)
+
+## General
+
+* {{breaking}} Upgrade required Java version to 22.0.2 and remove redundant JVM
```

Comment:
I think we actually dont need that now.

Reply:
See slack discussion https://trinodb.slack.com/archives/C07ABNN828M/p1721945210350989

Reply:
Yeah, there wasn't any change to the required version. We just updated the CI and docker image.